docs/pages/reference/runtime.md

```diff
@@ -0,0 +1,131 @@
+import Callout from 'nextra-theme-docs/callout'
+
+# Runtime
+
+**Benzene** supports GraphQL execution using a customizable runtime.
+
+Benzene's GraphQL runtime can be configured using the `compileQuery` option. By default, it will use [`graphql-js` implementation](#graphql-js).
```

Comment:
```suggestion
Benzene's GraphQL runtime can be configured with the `compileQuery` option. By default, it will use [`graphql-js`](#graphql-js) implementation.
```